tests/timeseries/test_sort_timestamps_monotonically.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import pandas as pd
+from random import randint
+from janitor.timeseries import sort_timestamps_monotonically
+
+
+# Random data for testing
+@pytest.fixture
+def timeseries_dataframe() -> pd.DataFrame:
+    """
+    Returns a time series dataframe
+    """
+    ts_index = pd.date_range("1/1/2019", periods=1000, freq="1H")
+    v1 = [randint(1, 2000) for i in range(1000)]
+    test_df = pd.DataFrame({"v1": v1}, index=ts_index)
+    return test_df
+
+
+@pytest.mark.timeseries
```

Comment:
I have this idea that the tests here can be parametrized, but I don't want to burden you further with this on this PR, as it does involve a bit more effort learning how to write parametrized tests (unless you're game for it!). Meanwhile, just leaving a suggestion here for documentation:

```suggestion
# NOTE: The tests possibly can be merged back together later if they are parametrized properly.

@pytest.mark.timeseries
```

Reply:
Cool stuff! I'm going to make another issue for this and then tag you on it, so that there's something we can use to remember!